studiopress/simple-social-icons
# Add Goodreads, Amazon, Reddit, and Tripadvisor icons

Adds icons for Goodreads, Amazon, Reddit, and Tripadvisor. I'm combining all the icons into 1 PR just to prevent merge conflicts.

<img width="199" alt="Screen Shot 2022-07-25 at 12 12 12 PM" src="https://user-images.githubusercontent.com/67485942/180836576-280410f8-b541-4190-abb8-92716922b940.png">




### How to test

1. Open the Simple Social Icons plugin on this branch
2. Add links to the form fields for Goodreads, Amazon, Reddit, and Tripadvisor
3. Check that the icons display correctly and they link to the correct place

### Documentation




### Suggested changelog entry

- Add Goodreads, Amazon, Reddit, and Tripadvisor icons

### Notes
